examples/platforms/nrf52840/diag.c

```diff
@@ -45,6 +45,7 @@
 #include <openthread/platform/radio.h>
 #include <openthread/platform/toolchain.h>
 
+#include <common/code_utils.hpp>
```

Comment:
The  `common/code_utils.hpp` is intended as internal to OT core header file (with C++ definitions which may not work in a C-99 source).  We have `platforms/utils/code_utils.h` for platform code to use. Please check this PR https://github.com/openthread/openthread/pull/1534.

Reply:
Thanks for the kind explanations. Will modify accordingly.